src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/CaseRepository.java

```diff
@@ -65,5 +66,7 @@ void saveCountyCourtJudgment(
     void saveReDetermination(String authorisation, Claim claim, ReDetermination reDetermination);
 
     void saveCaseEvent(String authorisation, Claim claim, CaseEvent caseEvent);
+
+    void linkClaimToDocument(String authorisation, Long claimId, ClaimDocumentStore claimDocumentStore);
```

Comment:
Prefer to rename this, as this is not linking a single document. How about `saveClaimDocuments(...)`?

Reply:
we link only one document at a time with the claim ..The reason I felt it was appropriate to use linkClaimToDocument was that the previous method was called linkSealedClaimDocument.

Reply:
ClaimDocumentStore is more than one document though isnt it?

Reply:
The method argument is not just one document, it's a collection of documents so it's no more linking one document here. Please rename.